src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -656,36 +656,12 @@ public Chat getChat() {
   }
 
   public static void main(final String[] args) {
-    System.out.println("Headless AWT Test: " + java.awt.GraphicsEnvironment.isHeadless());
-    handleCommandLineArgs(args);
-    // grab these before we override them with the loggers
-    final InputStream in = System.in;
-    final PrintStream out = System.out;
-    // after handling the command lines, because we use the triplea.game.name= property in our log file name
-    GameRunner.setupLogging(GameRunner.GameMode.HEADLESS_BOT);
+    GameRunner.handleCommandLineArgs(args, getProperties(), GameRunner.GameMode.HEADLESS_BOT);
     ClipPlayer.setBeSilentInPreferencesWithoutAffectingCurrent(true);
-    HeadlessGameServer server = null;
     try {
-      server = new HeadlessGameServer();
+      new HeadlessGameServer();
     } catch (final Exception e) {
-      ClientLogger.logQuietly(e);
-    }
-    if (Boolean.parseBoolean(System.getProperty(GameRunner.TRIPLEA_GAME_HOST_CONSOLE_PROPERTY, "false"))) {
```

Comment:
TRIPLEA_GAME_HOST_CONSOLE_PROPERTY triggers whether we show the bot UI. We have agreed we will stop launching bots in screen sessions and will no longer be using their UI. The majority of bots are not running with this configuration.